### DISCUSSION TITLE
How difficult is it to scrape another website like anicloud.io?

### DISCUSSION THREAD

**Discussion by speedyconzales** [pystardust/ani-cli#597] [Q&A] [answered]
Do you plan to expand the source of this cli? Scraping more sites than just gogoanime? And how difficult is this to accomplish?

**Comment by frostbyteninja** [comment, marked as answer]
Hello my name is FireTonnor I am planning to add more sources to add to the cli if you have suggestion please go to our discord server or go to this google form for you to know I am not the owner or maintain  the cli so if you put a suggestion on the form the I will respond and try to add the stuff you all send https://forms.gle/EkfwPGycEScBpqm98

**Comment by port19x** [comment]
At this point a rewrite for another site is tempting
